implementations/contracts/utils/ErrorHandlerLib.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+/**
```

Comment:
Suggestion to finish the comment description of the library in comment 🙂 

_before_
```
/**
 * @title Error Handler
 * @dev When calling other contracts using `contract.call{ ... }(data)` errors need to
 * handled manually. This library
 */
```

_after_

```
/**
 * @title Error Handler
 * @dev When calling other contracts via low-level calls like `contract.call{ ... }(data)`, errors need to
 * be handled manually. This library provides this functionality by bubbling up either revert reason strings,
 * or custom Errors introduced in Solidity 0.8.4
 */
```